Makefile

```diff
@@ -160,6 +160,9 @@ generate_go: bin/syz-sysgen format_cpp
 generate_sys: bin/syz-sysgen
 	bin/syz-sysgen
 
+generate_fidl:
+	$(GO) generate ./sys/fuchsia
+
```

Comment:
Please add here:
````
	$(MAKE) format_sys
```
Ideally it should not produce diff with what's currently checked in, and this will require re-formatting.